erizo/src/erizo/rtp/RtpPaddingGeneratorHandler.cpp

```diff
@@ -110,7 +111,9 @@ void RtpPaddingGeneratorHandler::onPacketWithMarkerSet(std::shared_ptr<DataPacke
   for (uint i = 0; i < number_of_full_padding_packets_; i++) {
     sendPaddingPacket(packet, kMaxPaddingSize);
   }
-  sendPaddingPacket(packet, last_padding_packet_size_);
+  // Temporary fix since https://bugzilla.mozilla.org/show_bug.cgi?id=1435025 is fixed
+  // Only send full rtp padding packet as suggested also in webrtc code.
+  // sendPaddingPacket(packet, last_padding_packet_size_);
```

Comment:
this could affect the way the congestion control works, have you tested it with Chrome too? did you notice any significant change in the amount of time it takes for simulcast to increase the quality layers?